### PR TITLE
Update FIFA World Cup trivia list with changes from 2022 tournament

### PR DESCRIPTION
**Pull request by Kreusada**
### Description of the changes

Updates the FIFA World Cup trivia list to include outcomes of the 2022 tournament as well as other changes to questions. I've added questions about player rewards, shock results, all-time nation stats, appearances, and more general questions.

If anyone has additional questions worthy of being added, leave a comment or let me know on Discord.

### Have the changes in this PR been tested?

Yes

### CODE REVIEW

File: redbot/cogs/trivia/data/lists/worldcup.yaml [hunk 1]
```diff
@@ -12,215 +12,221 @@ AUTHOR: Kreusada
 
 # Hosts
 
-Who are the hosts of the 2022 World Cup?:
+Which country were the hosts of the 2022 World Cup?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/worldcup.yaml]: `were the hosts of` -> `was the host of`
`were the champions of` -> `was the champion of`
`were the runner ups of` -> `was the runner up of`
For all questions of these formats

File: redbot/cogs/trivia/data/lists/worldcup.yaml [hunk 2]
```diff
@@ -432,3 +456,249 @@ In 2018, Italy failed to qualify after losing to *who* 1-0?:
 - Sweden
 In 2002, Netherlands failed to qualify after losing to Republic of Ireland. What was the goal difference in this qualification decider?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/worldcup.yaml]:
```suggestion
In 2002, Netherlands failed to qualify after losing to the Republic of Ireland. What was the goal difference in this qualification decider?:
```
While you're touching this

File: redbot/cogs/trivia/data/lists/worldcup.yaml [hunk 3]
```diff
@@ -432,3 +456,249 @@ In 2018, Italy failed to qualify after losing to *who* 1-0?:
...
+- Five
+Prior to Kylian Mbappé's hatrick in the final of 2022, who was the only player to have scored a hattrick in a FIFA World Cup final?:
+- Geoff Hurst
+- Hurst
+
+#### APPEARANCES
+
+Which is the most northerly European nation who has never qualified for a FIFA World Cup?:
+- Finland
+Iceland have 1 FIFA World Cup appearance, but in which year?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/worldcup.yaml]: This structure is a little wordy, maybe
`<Country> has had only 1 FIFA World Cup appearance. In what year did they appear?`

**Review by Flame442 — CHANGES_REQUESTED** [review]

**Comment by Kreusada** [comment]
Thanks! The necessary changes have been addressed.

**Review by Flame442 — APPROVED** [review]

**Pull request merged**